indico/modules/rb_new/util.py

```diff
@@ -541,3 +549,36 @@ def serialize_nonbookable_periods(data):
 
 def serialize_unbookable_hours(data):
     return [bookable_hours_schema.dump(d).data for d in data]
+
+
+def build_rooms_spritesheet():
+    def _resize(image):
+        return image.resize(ROOM_PHOTO_DIMENSIONS, Image.ANTIALIAS)
+
+    image_width, image_height = ROOM_PHOTO_DIMENSIONS
+    query = Room.query.filter(Room.photo)
+    room_count = query.count()
```

Comment:
just get `.all()` from the rooms, since you get them all later anyway. no need to do a separate query for the count